From b642c9b2ee02247e95a10fd07f1d3d610013105f Mon Sep 17 00:00:00 2001
From: Francisco Boni <boboniboni@gmail.com>
Date: Wed, 1 Dec 2021 04:44:47 -0300
Subject: [PATCH 3/3] Use clearlinux CFLAGS during build

---
 Configurations/10-main.conf | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Configurations/10-main.conf b/Configurations/10-main.conf
index b141be5..253dd5e 100644
--- a/Configurations/10-main.conf
+++ b/Configurations/10-main.conf
@@ -633,6 +633,8 @@ sub vms_info {
     },
     "linux-generic64" => {
         inherit_from     => [ "linux-generic32" ],
+        cflags           => add($ENV{'CFLAGS'}),
+        ex_libs          => "$ENV{'EX_LIBS'}",
         bn_ops           => "SIXTY_FOUR_BIT_LONG RC4_CHAR",
     },
 
@@ -734,6 +736,8 @@ sub vms_info {
         inherit_from     => [ "linux-generic32", asm("x86_elf_asm") ],
         cflags           => add(picker(default => "-DL_ENDIAN",
                                        release => "-fomit-frame-pointer")),
+        cflags           => add($ENV{'CFLAGS'}),
+        ex_libs          => "$ENV{'EX_LIBS'}",
         bn_ops           => "BN_LLONG",
     },
     "linux-aout" => {
-- 
2.34.0

